### PR TITLE
fix(support): add new support plans

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: faq/suppport.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  title: Scaleway Support 
+  description: Discover Scaleway Support plans. 
+content:
+  h1: Scaleway Support 
+  paragraph: Discover Scaleway Support plans.
+image: general.webp
+---
+
+## What is my current support plan?
+You can check your support plan on the support plan page within the Scaleway console. The free Basic support plan is included for all Scaleway Elements customers automatically.
```
Comment (nedaehm) [inline faq/suppport.mdx:12]: Maybe add a link to the support plan page?

File: faq/suppport.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Our customer excellence team can assist you in English and French, via a 24/7 livechat, ticket and/or phone, depending on your support plan level.
+
+## How do I create a support ticket?
+You can create and track your support tickets from the Scaleway console. [Follow this how to](/console/my-account/how-to/open-a-support-ticket/) for more details. 
+
+## How do I report abuse or suspected abuse coming from the Scaleway network?
+To report abuse or suspicion of abuse from our network, please open a new abuse report via abuse.scaleway.com. If relevant, feel free to include your logs.
+
+## I am a bronze plan subscriber. Can I keep my old plan? 
+Your plan remains unchanged until January 1, 2023. Past this date, the Bronze plan will be removed from our pages and you will automatically be moved to the Silver plan. If you do not wish to be subscribed to the silver plan, you can downgrade your support plan to basic.
```
Comment (nedaehm) [inline faq/suppport.mdx:39]:
```suggestion
Your plan remains unchanged until January 1, 2023. Past this date, the Bronze plan will be removed from our pages and you will automatically be moved to the Silver plan. If you do not wish to be subscribed to the Silver plan, you can downgrade your support plan to basic. 
```

File: faq/suppport.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+| Guaranteed response time | <1hour |
+| Team support | Dedicated Premium Team |
+| Custom Onboarding | - |
+| Quarterly technical review | - |
+| On demand architecture review | - |
+
+
+## I subscribed to the Silver Plan before October 1. Can I keep it unchanged?  
+
+Your plan remains unchanged until January 1st, 2023. Past this date, the new modalities of the silver plan will apply.
```
Comment (nedaehm) [inline faq/suppport.mdx:56]:
```suggestion
Your plan remains unchanged until January 1st, 2023. Past this date, the new modalities of the Silver plan will apply.
```

File: faq/suppport.mdx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+You can switch plans once a month.
+
+## I would like to upgrade my plan to get faster responses. Once I upgrade, how soon will I receive the benefits of my new plan? 
+When you change your plan, the upgrade is applied immediately.
+
+## If I change my support plan in the middle of the month, how will I be billed?
+You will be billed on a pro rata basis. For example, if you switch from the Silver plan to the Gold plan on the 20th of the month, you will be billed from the first to the 19th for the Silver plan. Starting from the 20th until the end of the month, you will be charged for the Gold plan.
+
+## How does the Gold plan pricing work?
+For the Gold plan, we will charge you whichever is greater, either €499 (excluding taxes) or 3% of your net spend. For example: If your net spend is €15,000, then 3% of €15,000 = €450. In this case, €499 is less than €450, so you will pay the fixed price at €499.
```
Comment (nedaehm) [inline faq/suppport.mdx:27]: The last sentence is a bit confusing

Comment (RoRoJ) [inline faq/suppport.mdx:27]:
```suggestion
For the Gold plan, we will charge you whichever is greater, either €499 per month (excluding taxes) or 3% of your net monthly spend. For example: If your net spend is €15,000, then 3% of €15,000 = €450. In this case, €499 is less than €450, so you will pay the fixed price at €499.
```

Comment (RoRoJ) [reply faq/suppport.mdx:27]: Yes, €499 isn't less than €450 ?

File: faq/suppport.mdx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+## Which language does the support team speak?
+Our customer excellence team can assist you in English and French, via a 24/7 livechat, ticket and/or phone, depending on your support plan level.
+
+## How do I create a support ticket?
+You can create and track your support tickets from the Scaleway console. [Follow this how to](/console/my-account/how-to/open-a-support-ticket/) for more details. 
+
+## How do I report abuse or suspected abuse coming from the Scaleway network?
+To report abuse or suspicion of abuse from our network, please open a new abuse report via abuse.scaleway.com. If relevant, feel free to include your logs.
+
+## I am a bronze plan subscriber. Can I keep my old plan?
```
Comment (RoRoJ) [inline faq/suppport.mdx:38]:
```suggestion
## I am a Bronze plan subscriber. Can I keep my old plan? 
```

File: faq/suppport.mdx (hunk 2 above)
Comment (RoRoJ) [inline faq/suppport.mdx:39]:
```suggestion
Your plan remains unchanged until January 1, 2023. Past this date, the Bronze plan will be removed from our pages and you will automatically be moved to the Silver plan. If you do not wish to be subscribed to the silver plan, you can downgrade your support plan to Basic. 
```

File: faq/suppport.mdx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+## I am a bronze plan subscriber. Can I keep my old plan? 
+Your plan remains unchanged until January 1, 2023. Past this date, the Bronze plan will be removed from our pages and you will automatically be moved to the Silver plan. If you do not wish to be subscribed to the silver plan, you can downgrade your support plan to basic. 
+
+## What will be included in my future Silver plan?
+
+| | New Silver Plan |
+:--: | :--: |
+| Pricing | 59€/month excluding taxes |
+| Dedicated assistance | Mon-fri From 9AM to 6PM |
```
Comment (RoRoJ) [inline faq/suppport.mdx:46]:
```suggestion
| Dedicated assistance | Mon-Fri from 9AM to 6PM |
```

File: faq/suppport.mdx (hunk 3 above)
Comment (RoRoJ) [inline faq/suppport.mdx:56]:
```suggestion
The pricing and content of your plan will remain unchanged until January 1st, 2023. After this date, the new Silver support plan package will apply, both in terms of pricing and content.
```

File: faq/suppport.mdx [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+## I subscribed to the Silver Plan before October 1. Can I keep it unchanged?  
+
+Your plan remains unchanged until January 1st, 2023. Past this date, the new modalities of the silver plan will apply.
+
+## What changes with the new Silver plan?
+|  | Old Silver Plan         | New Silver Plan |
+| ------------ | ------------------- | ------ |
+| Pricing        | 59€/month excluding taxes | 59€/month excluding taxes    |
+| Dedicated assistance    | Hotline 24/7      | Mon-fri From 9AM to 6PM    |
```
Comment (RoRoJ) [inline faq/suppport.mdx:62]:
```suggestion
| Dedicated assistance    | Hotline 24/7      | Mon-Fri from 9AM to 6PM    |
```

**Review by nedaehm — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**